plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -346,6 +347,19 @@ public DeltaLakeConfig setExtendedStatisticsEnabled(boolean extendedStatisticsEn
         return this;
     }
 
+    public boolean isCollectExtendedStatisticsColumnStatisticsOnWrite()
+    {
+        return collectExtendedStatisticsColumnStatisticsOnWrite;
+    }
+
+    @Config("delta.extended-statistics.collect-on-write")
```

Comment:
Do consider documenting this new property in `delta-lake.rst` - either in this PR or a follow-up PR

Reply:
I would wait with documentation until other write operations are implemented if that's ok.